src/components/Environments/List/EnvironmentListView.tsx

```diff
@@ -113,9 +118,11 @@ export default function EnvironmentsListView({ removeAllFilters }: EnvironmentsL
                                 {envData.environment_name}
                             </NavLink>
                         </div>
-                        <div>{envData.namespace}</div>
+                        <div className='dc__truncate-text'>{envData.namespace}</div>
                         <div>{envData.cluster_name}</div>
-                        <div>{envData.appCount || 0} {GROUP_LIST_HEADER.APPLICATIONS}</div>
+                        <div>
+                            {envData.appCount || 0} {GROUP_LIST_HEADER.APPLICATIONS}
```

Comment:
It should be 0 or 1 application & 2 applications, right?

Reply:
yes